Tasks/GooglePlayReleaseV4/modules/googleutil.ts

```diff
@@ -96,7 +98,10 @@ export async function updateTrack(
         release.releaseNotes = releaseNotes;
     }
 
-    if (userFraction < 1.0) {
```

Comment:
https://developers.google.com/android-publisher/api-ref/rest/v3/edits.tracks#status

Shows that the status enum has 5 possible values. I think allowing users to pass status for release might be more flexible. If they add more statuses, clients who would have used it wouldn't need to update. But non the less this is a great addition.